events/job_offmap.txt

```diff
@@ -1013,34 +1016,43 @@ character_event = {
 					}
 				}
 				random_list = {
-					10 = {
+					8 = {
 						set_character_flag = light_foot_leader
 					}
-					10 = {
-						set_character_flag = heavy_infantry_leader
```

Comment:
Put heavy_infantry_leader to its original place so that the GitHub will stop thinking that you removed it.